app/models/concerns/issues/grouped_export.rb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+module Issues::GroupedExport
+  extend ActiveSupport::Concern
+
+  CSV_OPTIONS       = { col_sep: ';' }
+  HEADERS_SEPARATOR = '⁻-⁻'
+
+  module ClassMethods
+    def grouped_export
+      csvs_by_script = {}
+      grouped_ids    = ids_grouped_by_script_id
+      scripts        = Script.where(id: grouped_ids.keys).pluck(:id, :name).to_h
+
+      grouped_ids.each do |script_id, issue_ids|
+        acc         = {}
+        script_name = scripts[script_id]
+
+        summarized_find_each_for_ids issue_ids do |issue|
+          issue.grouped_export script_name, acc
+        end
+
+        grouped_csvs = build_csvs_for_groups acc
+        filename     = sanitize_filename(script_name) + '.zip'
```

Comment:
Esto me pinta más en la forma:

```suggestion
        filename     = "#{sanitize_filename script_name}.zip"
```

Que hasta se presta para un Seattle _embebido_ =P